com.unity.netcode.gameobjects/Runtime/Metrics/ExposedMetricsHook.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using UnityEngine;
+
+namespace Unity.Netcode
+{
+    public class ExposedMetricsHook
+    {
+        public static ExposedMetricsHook Instance => s_Instance;
```

Comment:
I don't see this being initialized anywhere (s_Instance)

Reply:
this class is inited in NetworkManager
` #region OOI_CC
MessagingSystem.Hook(new ExposedMetricsHook().InternalMetricHook);
#endregion`